ipfs-cluster-ctl/formatters.go

```diff
@@ -81,7 +81,7 @@ func textFormatPrintIDSerial(obj *api.IDSerial) {
 		return
 	}
 
-	fmt.Printf("%s | Sees %d other peers\n", obj.ID, len(obj.ClusterPeers)-1)
+	fmt.Printf("%s | '%s' | Sees %d other peers\n", obj.ID, obj.Hostname, len(obj.ClusterPeers)-1)
```

Comment:
better not single-quote the alias. I'm thinking when running a silly grep + cut and and piping things it's causes an extra step to remove quotes